spec/support/vcr.rb

```diff
@@ -11,4 +11,11 @@
 
   # Only want VCR to intercept requests to external URLs.
   config.ignore_localhost = true
+
+  # Ignore any solr/fedora requests
+  config.ignore_request do |request|
+    if ENV['FCREPO_URL'] ||  ENV['SOLR_URL']
+      request.uri.include?(ENV['FCREPO_URL']) || request.uri.include?(ENV['SOLR_URL'])
```

Comment:
Now that fedora/solr are not on the same machine (when using docker), we need to tell VCR to ignore any traffic to fedora/solr